src/redux/actions/samples/createSamples.js

```diff
@@ -115,9 +118,13 @@ const createSamples = (
         options,
         metadata: experiment?.metadataKeys
           .reduce((acc, curr) => ({ ...acc, [curr]: METADATA_DEFAULT_VALUE }), {}) || {},
-        files: Object.values(files).reduce(((acc, curr) => (
-          { ...acc, [curr.name]: { upload: { status: UploadStatus.UPLOADING } } }
-        )), {}),
+        files: Object.values(files).reduce(((acc, curr) => {
+          const fileType = fileNameForApiV1[getFileTypeV2(curr.name, sampleTechnology)];
```

Comment:
why do we use a get file for v2, but then it' used to index file names for api v1? it's a bit confusing.

Reply:
I agree it's a bit confusing. There was small bug here and I fixed it by doing this. We need to use the `fileNameForApiV1` here, but the function that identifies file types is `getFileTypeV2`, so I had to use them in conjunction to end up with the v1 type. 

This is probably a good time to get rid of all this v1 file naming so I think I will do that refactor after I am done with these.